Samples/iOS-Swift/iOS-Swift-UITests/LaunchUITests.swift

```diff
@@ -102,7 +102,6 @@ class LaunchUITests: BaseUITest {
         XCTAssertEqual(app.state, .notRunning)
         
         app.launch()
-        XCTAssertEqual(app.state, .runningForeground)
```

Comment:
Removing this because checking for the app state right after closing the app can be flake and fail in CI.
`waitForExistence` in the next line has a 10 seconds timeout to wait for the app to open again.

Reply:
`h`: please test this against 8.33.0 and ensure the test still fails

Reply:
Yes it does fail.

Reply:
does it work and make sense to switch the lines, i.e. first `waitForExistence` and then the assertion that app is running

Reply:
It does work. Not sure if it makes a different since waitForExistence does need the app to be open, but I also dont think it will harm.

Reply:
then let's keep it pls